namerd/storage/k8s/src/test/scala/io/buoyant/namerd/storage/K8sConfigTest.scala

```diff
@@ -0,0 +1,39 @@
+package io.buoyant.namerd.storage
+
+import io.buoyant.config.Parser
+import io.buoyant.config.types.Port
+import io.buoyant.namerd.DtabStoreConfig
+import io.buoyant.test.FunSuite
+import org.scalatest.OptionValues
+
+class K8sConfigTest extends FunSuite with OptionValues {
+  test("parse config") {
+    val yaml =
+      """
+      |kind: io.l5d.k8s
+      |host: localhost
+      |port: 8001
+      |namespace: default
+    """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(K8sDtabStoreInitializer)))
+    val k8s = mapper.readValue[DtabStoreConfig](yaml).asInstanceOf[K8sConfig]
+    assert(k8s.host.value == "localhost")
+    assert(k8s.port.value == Port(8001))
+    assert(k8s.namespace.value == "default")
+  }
+
+  test("parse v2 config") {
```

Comment:
rm

Reply:
In terms of manual testing, I was using namerctl and minikube using both TPRs and CRDs. I mainly tested whether an update and create was successful. I am going to test delete to see if it works as well but I am confident that it will.